### DISCUSSION TITLE
log file output or misinterpreted the -l flag

### DISCUSSION THREAD

**Discussion by mckaygerhard** [filebrowser/filebrowser#1598] [Q&A]
i amd seraching for log to file.. i will not cited the OUT DATED documentation.. command line does not details:
`  -l, --log string                         log output (default "stdout")`

so the log only are standar output adn standar error, i can specify a file for? or i must use unix linux redirection 2&1 ??

**Comment by ramiresviana** [comment]
The log option accepts a path to a file or the `stdout` and `stderr` keywords. When given no parameter, it will disable logging.

**Reply by mckaygerhard** [reply]
i try with relative path and absolute one.. and this does not work...

![imagen](https://user-images.githubusercontent.com/1571724/136257954-bd554466-8491-4873-8d93-28b7edf68e16.png)

and still redirects to stdout.. but still no file generated.. after several tryes.. i must to force by absolute path, works but still redirect to stdout.. 

i try to setup in config file and import/export and it do not take the configuration

**Reply by mckaygerhard** [reply]
#1599 issued due that
